src/lib/pickles/wrap_verifier.ml

```diff
@@ -521,12 +649,18 @@ struct
         let index_digest =
           with_label "absorb verifier index" (fun () ->
               let index_sponge = Sponge.create sponge_params in
-              Array.iter
-                (Types.index_to_field_elements
+              List.iter
+                (index_to_field_elements
                    ~g:(fun (z : Inputs.Inner_curve.t) ->
                      List.to_array (Inner_curve.to_field_elements z) )
                    m )
-                ~f:(fun x -> Sponge.absorb index_sponge x) ;
+                ~f:(function
+                  | Plonk_types.Opt.None ->
+                      ()
+                  | Plonk_types.Opt.Maybe _ ->
+                      failwith "TODO"
```

Comment:
Could we maybe have a more extended documentation than `TODO`? I'm asking for a friend who's afraid he might need to fill the TODO in the future :-D